core/arch/arm/plat-versal2/conf.mk

```diff
@@ -34,7 +38,7 @@ CFG_WITH_STATS		?= y
 CFG_ARM64_core		?= y
 
 # Enable ARM Crypto Extensions(CE)
-$(call force,CFG_CRYPTO_WITH_CE,y)
+CFG_CRYPTO_WITH_CE ?= y
```

Comment:
This will not work as intended. If `CFG_CRYPTO_WITH_CE82` is `y`, then `CFG_CRYPTO_WITH_CE` must also be `y`.